File: java-mongo-app/JavaMongoApp/src/javamongoapp/connections/Connection.java
package javamongoapp.connections;

import com.mongodb.MongoClient;
import com.mongodb.client.MongoDatabase;

public class Connection {
    
     public static MongoDatabase Connect() {
        MongoClient client = new MongoClient("localhost", 27017);
        MongoDatabase db = client.getDatabase("dbSharp");
        return db;
    }
}

File: crud-sharp-mongodb/crudsharp-mongodb/libCrud/Connection.cs
using MongoDB.Driver;
using System;
using System.Collections.Generic;
using System.Data.SqlClient;
using System.IO;
using System.Linq;
using System.Text;

namespace libCrud
{
    public class Connection
    {
        public static IMongoDatabase Connect()
        {
            try
            {
                MongoClient client = new MongoClient("mongodb://localhost:27017");
                IMongoDatabase db = client.GetDatabase("dbSharp");
                return db;
            }
            catch (Exception)
            {
                return null;
            }
        }
    }
}
